CI failed on UI tests again, this time:

```
**********************************************************************
File "ui.rst", line ?, in default
Failed example:
    b"Paused" in p.recvuntil(b"press any")
Exception raised:
    Traceback (most recent call last):
      File "/opt/hostedtoolcache/Python/2.7.18/x64/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest default[3]>", line 1, in <module>
        b"Paused" in p.recvuntil(b"press any")
      File "/home/runner/work/pwntools/pwntools/pwnlib/tubes/tube.py", line 310, in recvuntil
        res = self.recv(timeout=self.timeout)
      File "/home/runner/work/pwntools/pwntools/pwnlib/tubes/tube.py", line 82, in recv
        return self._recv(numb, timeout) or b''
      File "/home/runner/work/pwntools/pwntools/pwnlib/tubes/tube.py", line 160, in _recv
        if not self.buffer and not self._fillbuffer(timeout):
      File "/home/runner/work/pwntools/pwntools/pwnlib/tubes/tube.py", line 131, in _fillbuffer
        data = self.recv_raw(self.buffer.get_fill_size())
      File "/home/runner/work/pwntools/pwntools/pwnlib/tubes/process.py", line 716, in recv_raw
        raise EOFError
    EOFError
**********************************************************************
```

--- a/pwnlib/ui.py
+++ b/pwnlib/ui.py
@@ -43,6 +43,7 @@ atexception.register(lambda:os.kill(os.getppid(), signal.SIGUSR1))
         raise EOFError("process terminated with code: %r (%r)" % (p.poll(True), p.stderr.read()))
     fcntl.ioctl(p.stdout.fileno(), termios.TIOCSWINSZ, struct.pack("hh", 80, 80))
     p.stdout.write(b"\x1b[1;1R")
+    time.sleep(0.5)
     return p
 
 def yesno(prompt, default=None):
@@ -308,9 +309,6 @@ def more(text):
       :const:`None`
 
     Tests:
-
-    .. doctest::
-       :skipif: github_actions
 
         >>> more("text")
         text

